remove-element.cpp

```diff
@@ -0,0 +1,28 @@
+// Author:Feyza Sarı
+// Question Link: https://leetcode.com/problems/remove-element/
+// O(n)
+
+/* First we have to check our element is equal to val.
+ If it is not nothiing change. But if they are equal we have to remove the
+ element. When we remove an element our array decrease one. So if we swap the
+ element - that equals to val - and the last element of array and decrease the
+ size of array we get the wanted array. */
+class Solution {
+public:
+    int removeElement(vector<int> &nums, int val) {
+        int i = 0;                  // current place
+        int size = nums.size() - 1; // array size
```

Comment:
bir de bu yorumları üstünde tutmalısın yanına yazmıyoruz
// array size
int size = nums.size() - 1;
gibi diğer satırlar için genel söylüyorum bunu